pkg/controller/vaultsecret/vaultsecret_controller.go

```diff
@@ -279,6 +279,11 @@ func (r *ReconcileVaultSecret) newSecretForCR(cr *maupuv1beta1.VaultSecret) (*co
 		labels[key] = val
 	}
 
+	annotations := map[string]string{}
```

Comment:
As no change is to necessary and metadata is also a `map[string]string`, it should be possible to directly assign it to the secret object below.

Reply:
Yes, that is way more elegant. Thanks